src/components/users/viewCohorts/components/NoCohortList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import '../viewCohort.css';
+import { Link } from 'react-router-dom';
+import { Box, List, ListItem, Avatar } from '@mui/material';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+
+export default function NoCohortList({ noCohort, addStudent }) {
+    return (
+        <Box className='Container_addStudent'>
+            <div className='box-title'>
+                <h3>Available students</h3>
+            </div>
+            <Box className='add-student-container'>
+                <List>
+                    {noCohort.map((student, key) => (
```

Comment:
There is a lot happening in this JSX. 

Could you please split it up into small components so that the JSX is a little easier to understand at a glance.